delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/model/Project.scala

```diff
@@ -46,11 +45,14 @@ final case class Project(
     organizationUuid: UUID,
     description: Option[String],
     apiMappings: ApiMappings,
+    defaultApiMappings: ApiMappings,
```

Comment:
Default API mappings are now injected here and not in the state as we don't want to persist them